PocketKit/Sources/PocketKit/Home/HomeViewModel.swift

```diff
@@ -97,6 +97,12 @@ class HomeViewModel: NSObject {
 
     @Published var tappedSeeAll: SeeAll?
 
+    var numberOfHeroItems: Int = 1 {
+        didSet {
+            self.snapshot = buildSnapshot()
```

Comment:
This works, but I have some concerns in updating the snapshot here because it might also happen at the same time inside `controller(_:didChangeContentWith:)` (perhaps because a refresh has been triggered?). On the other hand, I can't think of a solution that would not require major refactors to the way we render recommendations right now, so I'd say we can go ahead and test it thoroughly before we release.

Reply:
good call out! Haven't seen any issues when testing but will keep this in mind. With the implementation of Unified Home, we may be able to do the refactor then.